src/main/java/cubicchunks/debug/DebugWorldType.java

```diff
@@ -108,8 +108,8 @@ public WorldProvider getReplacedProviderFor(WorldProvider provider) {
 			}
 
 			@Override
-			public Vec3i[] getPopRequirment(Cube cube) {
-				return NO_POPULTOR_REQUIRMENT;
+			public Vec3i[] getPopRequirement(Cube cube) {
```

Comment:
I'd still prefer if it was spelled out - `getPopulationRequirement` or `getPopulatorRequirement`. Also should maybe be plural (requirements)?